frontend/src/components/groups/Group.tsx

```diff
@@ -92,6 +96,9 @@ export function Group() {
 							<RoleChip role={role}/>
 						</Box>
 						<Typography variant="body1" paragraph>{groupAbout.description}</Typography>
+						<Typography variant="body1" paragraph><strong>Creator: </strong><ListItem>
+   							<a href={groupCreatorEmailLink} target="_top">{groupCreatorEmail}</a>
```

Comment:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/13950475/233376006-d973999c-5ebb-40c8-ab3a-e7fbdf458ddc.png">
Instead of plain <a> Tag, can we use either MUI <Link> tag so it turns orange :-)